src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class AbstractFigure implements Drawable, AreaCalculator {
+    private String type;
+    private String color;

```

Comment:
since we have an enum `Color` let's use it here as a type, it is always preferred to avoid raw types like string and use their object representation instead